OpenImis.RestApi/wwwroot/SentMessages/PaymentConfirmationSms_255744555686353670.json

```diff
@@ -0,0 +1 @@
+[{"Message":"Payment has been made to CHF- Invoice: 995940004307 Amount: 1780 TZS Receipt: 92116200000xx938 Date: 28/09/2021 15:58:31 Through:1623416311901","Recipient":"255744555686","Response":"\n{\"error\":false,\"statusCode\":48200,\"statusMessage\":\"Success\",\"data\":[\"Message accepted delivery,Balance:927065\"]}____OK"}]
```

Comment:
All messages from SendMessages to be removed from PR. Add an exception in gitignore.

Reply:
Done!